lib/mllp/receiver.ex

```diff
@@ -171,7 +171,14 @@ defmodule MLLP.Receiver do
       Keyword.get(opts, :dispatcher, nil) ||
         raise(ArgumentError, "No dispatcher module provided")
 
+    Code.ensure_loaded?(dispatcher_mod) ||
```

Comment:
Should we also check for if it implements behaviour `MLLP.Dispatcher`

Reply:
Yup :) Excellent call out!